src/libraries/System.Security.Cryptography.Cng/src/System/Security/Cryptography/CngKey.Export.cs

```diff
@@ -91,15 +91,16 @@ public byte[] Export(CngKeyBlobFormat format)
             int kdfCount)
         {
             bool ret = ExportPkcs8KeyBlob(
-                true,
+                allocate: true,
                 _keyHandle,
                 password,
                 kdfCount,
                 Span<byte>.Empty,
                 out _,
-                out byte[] allocated);
+                out byte[]? allocated);
 
             Debug.Assert(ret);
+            Debug.Assert(allocated != null); // on account of allocate parameter being true
```

Comment:
Isn't it redundant on account of `allocate: true`? Seems its always init-ed in that case